### PR TITLE
Change text-domain

### PR DESCRIPTION
**Pull request by kebbet**
Solves #19 
Made a search and replace. Have not tested it.

### CODE REVIEW

**Comment by kebbet** [comment]
The second commit tested on live site with the translations available.  So the translations that has been made will hopefully work! 

image

**Comment by soup-bowl** [comment]
Thanks for the PR! I can verify on my end all the text-domain and language changes work.

There's been a couple of changes to URLs in your submit, which seems to break the test dispatch and the log URIs. Was there an intention to change the URLs, or was it because the path names matched the text domains?

The changes occur:
- class-logtable.php, line 154, and 182-184.
- class-settings.php line 415.

If this was intentional, there's only a few more places the change (wpsimplesmtp to wp-simple-smtp) would be needed:
- class-logtable.php, line 132 (used as the base URL for the log buttons).
- class-settings.php, line 226 (return user after test email is sent).

**Comment by kebbet** [comment]
The URL changes was unintentional, I'll correct and commit again.

File: class-settings.php [hunk 1]
```diff
@@ -113,10 +113,10 @@ public function options_page() {
 	 */
 	public function add_admin_menu() {
 		add_options_page(
-			__( 'Mail', 'wpsimplesmtp' ),
-			__( 'Mail', 'wpsimplesmtp' ),
+			__( 'Mail', 'simple-smtp' ),
+			__( 'Mail', 'simple-smtp' ),
 			'manage_options',
-			'wpsimplesmtp',
+			'wp-simple-smtp',
```
Comment (soup-bowl) [inline class-settings.php:119]: One more, apologies I've missed this from my review.

Comment (kebbet) [reply class-settings.php]: Just found it myself while testing, no worries :)

**Comment by soup-bowl** [comment]
Perfect, thank you for your commit! I'll prep to release this as a bug fix soon.

**Comment by kebbet** [comment]
> Perfect, thank you for your commit! I'll prep to release this as a bug fix soon.

You're welcome, it was fun to do my first ever PR on a GitHub-project :D

**Comment by soup-bowl** [comment]
Wow that was your first? Nice one! You'd been by far one of the most collaborative PR's I've had, best of luck on your GitHub adventures :smile:

**Pull request merged**